src/client/Microsoft.Identity.Client/MsalClaimsChallengeException.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.TelemetryCore.Internal.Events;
+using static Microsoft.Identity.Client.TelemetryCore.Internal.Events.ApiEvent;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// This exception indicates that claims have been returned from the endpoint that need to be resolved. This can be achieved by using them in the next authentication request.
+    /// See https://learn.microsoft.com/en-us/entra/msal/dotnet/advanced/exceptions/msal-error-handling#conditional-access-and-claims-challenges
+    /// See https://learn.microsoft.com/en-us/entra/msal/dotnet/acquiring-tokens/web-apps-apis/on-behalf-of-flow#handling-multi-factor-auth-mfa-conditional-access-and-incremental-consent
```

Comment:
These can be put into an href, so they're clickable.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/57e840463445e5760b49388a5b3ba0af27eb94e2/src/client/Microsoft.Identity.Client/LogLevel.cs#L8

Reply:
Also, as @pmaytak mentioned in the meeting, remove the `/en-us/` identifiers - locales are auto-inferred when the customer navigates to the page.